conf/paths.go

```diff
@@ -12,6 +12,7 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
+//go:build !local
```

Comment:
We have now added these back :shrug:

Reply:
Moth*&*%(*&%,

Reply:
Man, do `git add -p` and review patch by patch what you are adding to your commit :)

Reply:
Yeh, lesson learnt here. Will stop trusting the tooling :see_no_evil: 
sry

Reply:
Trust, but verify :heavy_check_mark:

Reply:
haha, alright.
Manually staged every chunk, scrolled through it in GitHub.
Will commence to buy coffee in case of discovered errors.